include/aspect/compat.h

```diff
@@ -81,4 +84,14 @@ namespace big_mpi
 
 }
 
+#if DEAL_II_VERSION_GTE(9,5,0)
+
+using dealii::VectorTools::compute_no_normal_flux_constraints_on_level;
+
+#else
+
+using aspect::internal::TangentialBoundaryFunctions::compute_no_normal_flux_constraints_on_level;
```

Comment:
instead of this, put the implementation here:
```
namespace dealii
{
namespace VectorTools
{
void compute_no_normal_flux_constraints_on_level(...)
...
```